iroha/src/config.rs

```diff
@@ -25,7 +25,8 @@ const IROHA_PRIVATE_KEY: &str = "IROHA_PRIVATE_KEY";
 const COMMIT_TIME_MS: &str = "COMMIT_TIME_MS";
 const TX_RECEIPT_TIME_MS: &str = "TX_RECEIPT_TIME_MS";
 const DEFAULT_TORII_URL: &str = "127.0.0.1:1337";
-const DEFAULT_BLOCK_TIME_MS: u64 = 1000;
+/// Amount of time peer waits for the `CreatedBlock` message after getting a `TransactionReceipt`
+pub const DEFAULT_BLOCK_TIME_MS: u64 = 200;
```

Comment:
hide it

Reply:
The defaults are used in some of the sumeragi tests, that's why I had to make them public.